src/fixate/reporting/csv.py

```diff
@@ -115,45 +115,54 @@ class CSVWriter:
     def __init__(self):
         self.csv_queue = Queue()
         self.csv_writer = None
-        self.reporting = CsvReporting()
+
+        self.exception_in_test = False
+        self.failed = False
+        self.chk_cnt = 0
+        self.csv_path = ""
+        self.test_module = None
+        self.start_time = None
+        self.current_test = None
+        self.data = fixate.config.get_config_dict()
+        self.data.update(fixate.config.get_plugin_data("plg_csv"))
+        self.exception = None
+
+        self._topics = [
+            (self.test_start, "Test_Start"),
+            (self.test_comparison, "Check"),
+            (self.test_exception, "Test_Exception"),
+            (self.test_complete, "Test_Complete"),
+            (self.sequence_update, "Sequence_Update"),
+            (self.sequence_complete, "Sequence_Complete"),
+            (self.user_wait_start, "UI_block_start"),
+            (self.user_wait_end, "UI_block_end"),
+        ]
```

Comment:
This is good, but it needs `(self.driver_open, "driver_open")`

Reply:
Yeah it does... For some reason I thought you wanted me to drop it.
Back in now:
![image](https://github.com/user-attachments/assets/68572971-2bee-42fc-b7cb-6ee839238d33)